lunes_cms/cms/models/feedback.py

```diff
@@ -0,0 +1,34 @@
+# from django.contrib.contenttypes.fields import GenericForeignKey
+from django.contrib.contenttypes.models import ContentType
+from django.utils.translation import ugettext_lazy as _
+from django.db import models
+
+
+class Feedback(models.Model):
+    """
+    Contains a title, a description and a reference to the matter of feedback
+    """
+
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    discipline = models.ForeignKey(ContentType, on_delete=models.CASCADE)
+    document = models.ForeignKey(ContentType, on_delete=models.CASCADE)
+    training_set = models.ForeignKey(ContentType, on_delete=models.CASCADE)
+    # options = models.GenericForeignKey("general", discipline, document, training_set)
+    # matter = models.CharField(max_length=12, choices=options, default=options[0])
+
+    def __str__(self):
+        """String representation of Feedback instance
+
+        :return: title of feedback instance
+        :rtype: str
+        """
+        return self.title
+
+    class Meta:
+        """
+        Define user readable name of Feedback
+        """
+
+        verbose_name = _("feedback")
+        verbose_name_plural = _("feedback")
```

Comment:
```suggestion
        #: The plural verbose name of the model
        verbose_name_plural = _("feedback entries")
```
I'd go with e.g. "4 feedback entries" for the plural of feedback, otherwise "4 feedback" looks a bit strange.